easybuild/easyconfigs/p/ParaView/ParaView-5.4.1-intel-2017b-mpi.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'CMakeMake'
+
+name = 'ParaView'
+version = '5.4.1'
+versionsuffix = '-mpi'
+
+homepage = "http://www.paraview.org"
+description = "ParaView is a scientific parallel visualizer."
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % download_suffix]
+sources = ["ParaView-v%(version)s.tar.gz"]
+checksums = ['390d0f5dc66bf432e202a39b1f34193af4bf8aad2355338fa5e2778ea07a80e4']
+
+dependencies = [
+    ('X11', '20171023'),
+    ('Mesa', '17.2.4'),
+    ('libGLU', '9.0.0'),
+    ('Qt', '4.8.7'),
```

Comment:
switch to `Qt5` for paraview?

Reply:
I guess we could, but I'd rather stick to the same Qt for both `ParaView` & `OpenFOAM`...